clang/lib/CIR/CodeGen/CIRGenOpenACCRecipe.cpp

```diff
@@ -89,7 +89,7 @@ mlir::Value OpenACCRecipeBuilderBase::makeBoundsAlloca(
   std::transform_inclusive_scan(
       resultTypes.begin(), resultTypes.end(),
       std::back_inserter(allocasLeftArr), std::plus<bool>{},
-      [](QualType ty) { return !ty->isConstantArrayType(); });
+      [](QualType ty) { return !ty->isConstantArrayType(); }, false);
```

Comment:
Huh, looks like someone's library is missing overload 1 here :)

https://en.cppreference.com/w/cpp/algorithm/transform_inclusive_scan.html